tests/integration/test_models_vision_text_config_registry.py

```
@@ -132,7 +132,9 @@ def test_initialize_raises_on_connection_error(self) -> None:
         with patch("file_organizer.models.text_model.OLLAMA_AVAILABLE", True):
             model = TextModel(config)
 
-        with patch("ollama.Client", side_effect=ConnectionError("refused")):
+        with patch(
+            "file_organizer.models.text_model.ollama.Client", side_effect=ConnectionError("refused")
+        ):
             with pytest.raises(ConnectionError):
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)